PendingReleaseNotes.md

```diff
@@ -4,3 +4,7 @@
 
 
 ## Features
+
+### Ceph
+
+- Add the ability to create Ceph Filesystem SubVolumeGroup via a new CRD, refer to the [configuration](Documentation/ceph-fs-subvolumegroup.md) for more details
```

Comment:
This needs to go back to 1.8, right? These pending release notes would only be for 1.9